sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/src/IServiceRequestHandler.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.Azure.WebPubSub.AspNetCore
+{
+    /// <summary>
+    /// Service request handler.
+    /// </summary>
+    public interface IServiceRequestHandler
```

Comment:
Should this be an abstract class to allow adding new members without breaking changes?